packages/rum-recorder/src/domain/rrweb-snapshot/snapshot.spec.ts

```diff
@@ -73,3 +73,52 @@ describe('absolute url to stylesheet', () => {
     expect(absoluteToStylesheet(`url('')`, href)).toEqual(`url('')`)
   })
 })
+
+describe('serializeNodeWithId', () => {
```

Comment:
cf BS unit tests, adding those tests expose some IE incompatibility in snapshot code (array.from)